### ISSUE TITLE
cdata-section-elements attribute in xsl:output does not seem to be respected in pipeline-step

### ISSUE THREAD

**Issue by siebej** [Armatiek/xslweb#10]
I have declared a cdata-section-elements attribute and also declared output:serialization-parameters. When I use this in stand-alone transformation using saxon-he 10.5 (i.e. using <oXygen/> XML Developer), the elements mentioned  in the cdata-section-elements attribute are nicely put into a `<![CDATA[the content of the element mentioned  in the cdata-section-elements attribute is here]]> `

When I use this same xslt in a pipeline-step, they don't end up in a  cdata.

**Comment by maartenkroon** [comment]
Hi Siebe,

Attributes that are specified on an xsl:output element (like "indent", "encoding" and indeed "cdata-section-elements") are instructions to the serializer (from tree to string). In a transformation that does not serialize the result to a string (like the intermediate steps in an XSLWeb pipeline, which transform directly to a new "TinyTree"), the xsl:output attributes are completely ignored by Saxon. 

Keep in mind that XSLWeb always performs an additional stylesheet transformation after the last transformation in the pipeline (home/common/xsl/system/response/response.xsl). That is why XSLWeb copies all xsl:output attributes from your **last** transformation step in the pipeline to this additional transformation. 

In short, **xsl:output attributes (including cdata-section-elements) must be set on the last transformation step of your pipeline to effect the final serialized result**. I just tested this with the cdata-section-elements attribute and it works. If it doesn't work in your case please provide a small sample in which the problem can be reproduced. 

Maarten
